frontend/src/pages/Landing.tsx

```diff
@@ -1,17 +1,79 @@
 import React from "react";
-import { Button } from "@mantine/core";
+import { Text, Button } from "@mantine/core";
+import { FaArrowRight, FaAngleDown } from "react-icons/fa6";
+import background from "../assets/background_laptop_s.png";
+import landing_logo from "../assets/landing_logo.png";
+import "./Landing.css";
+import { Link } from "react-router-dom";
+import Navbar from "../components/Navbar";
 
 function Landing() {
   return (
-    <div>
-      {/* Example Usage for Mantine Components */}
-      <Button variant="filled" size="xl" radius="md">Button</Button>
-      <h1>Landing Page xD</h1>
-      <p>Type /route into link to go the page u wanna go to :P</p>
-      <a href="https://docs.google.com/document/d/1iFcgbbb_5nNqy9EMgA2oaZ2ZTpvSof95XA_n9mv16kI/edit">
-        <button>Click here to see routes</button>
-      </a>
-    </div>
+    <>
+      <div
+        className="landing-container"
+        style={{
+          width: "100%",
+          height: "100%",
+        }}
+      >
+        <div
+          className="hero-container"
+          style={{
```

Comment:
No inline styles in main plz haha.

Also overriding your styles from Landing.css